src/main/scala/no/ndla/conceptapi/service/search/ConceptSearchService.scala

```diff
@@ -32,61 +33,56 @@ trait ConceptSearchService {
     override def hitToApiModel(hitString: String, language: String): api.ConceptSummary =
       searchConverterService.hitAsConceptSummary(hitString, language)
 
-    def all(withIdIn: List[Long],
-            language: String,
-            page: Int,
-            pageSize: Int,
-            sort: Sort.Value,
-            fallback: Boolean): Try[SearchResult[api.ConceptSummary]] =
-      executeSearch(withIdIn, language, sort, page, pageSize, boolQuery(), fallback)
-
-    def matchingQuery(query: String,
-                      withIdIn: List[Long],
-                      searchLanguage: String,
-                      page: Int,
-                      pageSize: Int,
-                      sort: Sort.Value,
-                      fallback: Boolean): Try[SearchResult[api.ConceptSummary]] = {
-      val language = if (searchLanguage == Language.AllLanguages) "*" else searchLanguage
+    def all(settings: SearchSettings): Try[SearchResult[api.ConceptSummary]] = executeSearch(boolQuery(), settings)
+
+    def matchingQuery(query: String, settings: SearchSettings): Try[SearchResult[api.ConceptSummary]] = {
+      val language = if (settings.searchLanguage == Language.AllLanguages) "*" else settings.searchLanguage
 
       val titleSearch = simpleStringQuery(query).field(s"title.$language", 2)
```

Comment:
Skulle gjerne hatt fuzzy søk på tittel slik at du får fleire treff når du skal sette inn forklaring i artikkel.

Reply:
`simpleStringQuery` gjør allerede noen "fuzzy"-ting. Kikket litt på å bytte den ut med en `match`-type query som støtter mer fuzziness, men disse støtter ikke operatorer som `+`,`-` og `"` som jeg tror er viktigere for oss.